joomla/joomla-cms
# [4.1] Fix checked_out value in 4.1.0-2021-11-26.sql and rename to 4.1.0-2021-12-29.sql

Pull Request for [Issue #](https://github.com/joomla/joomla-cms/pull/36190#issuecomment-1002566708) and https://github.com/joomla/joomla-cms/pull/36190#issuecomment-1002569550 .

### Summary of Changes

Fix the value of the checked_out column in the not yet released update SQL script from PR #36190 .

Rename the script so that it runs when updating from 4.1-beta1 to the future 4.1-beta2 which will contain the changes from PR #36190 .

With the name "4.1.0-2021-11-26.sql" it will not run because the schema version in database of a 4.1-beta1 is already "4.1.0-2021-11-28.sql".

@bembelimen You should have noticed that when merging.

### Testing Instructions

1. On a 4.1-dev or recent 4.1 nightly build **_which does not yet include the changes from PR_** #36190 , run the update SQL statement from that PR, i.e. for MySQL the one here: https://github.com/joomla/joomla-cms/blob/4.1-dev/administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-11-26.sql (of course use the right table name prefix).
2. Check in the system dashboard if there are checked out extensions, and in com_checkout for which table they are.
Result: See section "Actual result BEFORE applying this Pull Request" below.
3. Delete the record in database which you have added in step 1.
4. Run the SQL statement from the modified update SQL of this PR here (of course use the right table name prefix), e.g. for MySQL:
````
INSERT INTO `#__extensions` (`name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
('plg_system_jooa11y', 'plugin', 'jooa11y', 'system', 0, 1, 1, 0, 1, '', '', '', NULL, NULL, 0, 0);
````
Result: See section "Expected result AFTER applying this Pull Request" below.

### Actual result BEFORE applying this Pull Request

![2021-12-29_checked-out-extension_1](https://user-images.githubusercontent.com/7413183/147661388-318fa672-3269-43de-8cc3-40828c5082f8.png)

![2021-12-29_checked-out-extension_2](https://user-images.githubusercontent.com/7413183/147661396-0c441b5b-c3ad-4995-a5df-83ebfece166c.png)

### Expected result AFTER applying this Pull Request

![2021-12-29_checked-out-extension_ok](https://user-images.githubusercontent.com/7413183/147661407-d4a823ab-db40-46c3-82a5-51e025280ff3.png)

### Documentation Changes Required

None.
